ccd-definition/AuthorisationCaseField.json

```diff
@@ -102,7 +102,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "documents_socialWorkOther",
     "UserRole": "caseworker-publiclaw-solicitor",
-    "CRUD": "CRU"
+    "CRUD": "CRUD"
```

Comment:
whoops, this change was pushed as well. Please revert to CRU. The change will be addressed in FPL-544.